src/main/java/forestry/farming/ModuleFarming.java

```diff
@@ -242,28 +242,28 @@ public void doInit() {
 
 		Circuits.farmArborealManaged = new CircuitFarmLogic("managedArboreal", arborealFarm, false);
 		Circuits.farmArborealManual = new CircuitFarmLogic("manualArboreal", arborealFarm, true);
-		arborealFarm.registerSoil(new ItemStack(Blocks.DIRT), coreBlocks.humus.getDefaultState());
-		arborealFarm.registerSoil(new ItemStack(coreBlocks.humus), coreBlocks.humus.getDefaultState());
+//		arborealFarm.registerSoil(Blocks.DIRT);	//TODO check this doesn't allow players to use dirt as soil in managed farm
```

Comment:
I'm not really sure how this worked before. Perhaps I can't simplify the API here

Reply:
I think it makes more sense to just keep `registerSoil(ItemStack, BlockState)` so other mods could register some sort of item and place a block that is completely unconnected to the item.

Reply:
Ok, I've kept the old API